src/app.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const firstFilePath = path.join(__dirname, '/file.txt');
+const secondFilePath = path.join(__dirname, '/file-copy.txt');
+
+const copyFiles = (file, fileToCopy) => {
+  if (file !== fileToCopy) {
+    const firstFileData = fs.readFileSync(file, 'utf-8');
+
+    fs.writeFileSync(fileToCopy, firstFileData);
+  } else {
```

Comment:
It would be more effective to throw an error instead of returning a string whenever someone tries to copy a file to the same location. This way, the error can be caught and handled appropriately in the calling scope.